docs-v2/docusaurus.config.js

```diff
@@ -51,11 +51,11 @@ const config = {
             from: '/tutorials.html',
           },
           {
-            to: '/docs/creating-charts-dashboards/first-dashboard',
+            to: '/docs/Creating Charts and Dashboards/creating-your-first-dashboard',
```

Comment:
not 100% sure what the change is here, but it's kinda odd to have spaces in a url. If this is replacing the old docs, then we should also make sure that redirects are setup between old and new urls

Reply:
I agree that it's weird to have space in the URL but that's where the new page is currently sitting. Not sure what do you mean about this though > then we should also make sure that redirects are setup between old and new urls. This is the scope of the redirect plugin. Here I am just making sure the links that it redirects to are correct, which they weren't in some cases.

Reply:
I agree that having spaces in the path is funny. I think it would be preferable to rename the paths so that they are kebab or some other non-space notation, but maybe that can be left to a follow up?

Reply:
Let's merge this PR which is unrelated to the way the URLs got created. We can discuss this separately and refactor the URLs in a separate PR.